Makefile

```diff
@@ -17,6 +17,15 @@
 VERSION ?= $(shell git describe --tags --always --dirty)
 LDFLAGS ?= -X main.version=$(VERSION)
 
+# Download portmap plugin
+download-portmap:
```

Comment:
We currently [bake CNI binaries in our AMI](https://github.com/awslabs/amazon-eks-ami/blob/9736e73/install-worker.sh#L78-L86), but we should really put this process in an init container (See #158).

Can you remove this for now?